scripts/functions.js

```diff
@@ -1,66 +1,69 @@
-
-
 const API_URL = "https://api.github.com/orgs/stackbuilders/repos";
 
-
 const getData = () => {
-
- return fetch(API_URL).then((response) => response.json());
-
-}
+  return fetch(API_URL).then((response) => response.json());
+};
 
 // FUCNTIONS TO RETURN THE REPOS WIHT MORE THAN FIVE STARS
 
 const getFilteredsStargazers = (data) => {
+  console.log(data);
+  return data.filter((repo) => repo.stargazers_count > 5);
+};
 
-  return data.filter(repo => repo.stargazers_count > 5)
-}
-
-
-const getReposMoreThanFiveStars = () => { 
-
-  return getData().then(data => {
+const getReposMoreThanFiveStars = () => {
```

Comment:
Make the functions inside this file pure, please.

Reply:
Could I just delete this file, because I think that the functions would be the same as the ones in the functionsAsyncAwait.js file.